### PR TITLE
applying pep8 character limit to the code Issue Issue #13 clean code in zip.py

### PR DESCRIPTION
**Pull request by saisiddhant12**
applying pep8 character limit to the code

Next Steps
1. Break the code into adding more number of functions
2. Include a config.yaml file 
3. Build class & Objects for the entire functionality

### CODE REVIEW

File: CONFIG/config.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+network:
+  subnet:
+    first_sheet:
```
Comment (achoudh5) [inline CONFIG/config.yml:3]: What does this signify?

File: CONFIG/config.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            98, 99, 100, 101, 102, 103, 104, 108]
+      c_grey_a: [42, 43, 120, 121]
+      c_yellow_a: [11, 12, 89, 90]
+      d_green_41_95_start_a: 128
+      d_green_41_95_end_a: 255
+      d_grey_44_start_a: 0
+      d_grey_44_end_a: 159
+      d_grey_122_start_a: 0
+      d_grey_122_end_a: 95
+    second_sheet:
```
Comment (achoudh5) [inline CONFIG/config.yml:16]: What does this signify?

**Comment by achoudh5** [comment]
@saisiddhant12 good stuff but I want you to think about where we will be using `config.yaml`. Will it be used for our Docker application or it can be used when we run `zip.py`? Also, made some comments on the PR!

**Review by achoudh5 — CHANGES_REQUESTED** [review]
Please check suggestions for`CONFIG/config.yml`

`zip.py` looks decent to me :)

File: CONFIG/config.yml (hunk 1 above)
Comment (saisiddhant12) [reply CONFIG/config.yml:3]: for a particular network in which we can have multiple subsets, and each subnet can have 2 or more sheets

File: CONFIG/config.yml (hunk 2 above)
Comment (saisiddhant12) [reply CONFIG/config.yml:16]: these are the hardcoded values taken from program as it is

**Comment by saisiddhant12** [comment]
> @saisiddhant12 good stuff but I want you to think about where we will be using `config.yaml`. Will it be used for our Docker application or it can be used when we run `zip.py`? Also, made some comments on the PR!

@achoudh5 we will use this as part of constructor while building class as each object will have this set of value

**Comment by saisiddhant12** [comment]
> > > @saisiddhant12 good stuff but I want you to think about where we will be using `config.yaml`. Will it be used for our Docker application or it can be used when we run `zip.py`? Also, made some comments on the PR!
> > 
> > 
> > @achoudh5 we will use this as part of constructor while building class as each object will have this set of value
> 
> @saisiddhant12 can you write a line or two with example of your thought process? I want to simplify it for anyone who later on picks up on this so they don't have to repeat the same thing! Let me know if it makes sense :)

> @achoudh5 definitely I will document it

**Comment by saisiddhant12** [comment]
### Process to consume the config file
The idea behind introducing a config file is to have the following features

1. Any change to the data, source of data will be done over here without disturbing the codebase
2. While creating an object of the class, these values will be initiated in default constructor __init__(), so that any instance of the class will have these as defaults and can be over ridden if required

Example to consume

classs network_alloc():
     def __init__(self):
           with open(.config.config.yml) as config_file:
                     self.object_config = yaml.load(config_file, Loader=yaml.FullLoader)
           self.c_green_a = self.object_config['network']['subnet']['first_sheet']['c_green_a']
           self.c_grey_a = self.object_config['network']['subnet']['first_sheet']['c_grey_a']

**Comment by achoudh5** [comment]
Thanks for your contribution @saisiddhant12

**Pull request merged**